plugins/setup.py

```diff
@@ -25,7 +25,8 @@
             'distributor = pulp_rpm.plugins.distributors.yum.distributor:entry_point',
             'ExportDistributor = pulp_rpm.plugins.distributors.export_distributor.distributor:'
             'entry_point',
-            'IsoDistributor = pulp_rpm.plugins.distributors.iso_distributor.distributor:entry_point'
+            'IsoDistributor = pulp_rpm.plugins.distributors.iso_distributor.distributor:entry_point',  # noqa
```

Comment:
change on line 28 is just the comma at the end of line
change on line 29 add the rsync distributor. Which makes me realize it should the RPMRsyncDistributor. I am naming the other one that will live in this project ISORsyncDistributor.